### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#55861]
#### VFS team name
VA Online Scheduling (VAOS)

#### VFS product name
VA Online Scheduling

#### VFS feature name
Appointment List

#### Point of Contact/Reviewers
Allison Christman - @allison0034 - Design

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
The "Go to your appointments" is styled as a button but it is navigation

### Link, screenshot or steps to recreate
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/42025-Allison-040701.png

### VA.gov Experience Standard
[Category Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Navigation should be links and not buttons. Change to a link or action link

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by ldelacosta** [comment]
@shiragoodman - We reviewed this ticket and determined that VAOS does not own or manage this page/code. This ticket may need Public Website assistance to review. If you have any questions, please reach out. Thanks! cc: @JRRoof

**Comment by shiragoodman** [comment]
thanks @ldelacosta. In the future, please just let us know so can have the ticket reassigned opposed to just closing it.

@wesrowe could you take a look at this issue to see if it is your teams responsibility? @allison0034 FYSA

**Comment by laflannery** [comment]
@wesrowe  this seems similar to the other [ticket here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/55859#issuecomment-1564967148), where you mentioned that it is going to be handled in the redesign that is underway. Do you know if this would also be updated at that time? Because if so I would recommend we handle this the same way - hold off for now and let this roll into the redesign

**Comment by outerpress** [comment]
@wesrowe @laflannery just checking in on this very old ticket as we try to clean up our board - not urgent on our end. Any plans on your side to update the login widgets to change the button to a link? The Cerner redirect was done but the login widget is still showing the issue. (If this is something our team should be doing, let me know. Thanks!)

<img width="1476" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL05WQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--f3787c59baf453da25bb8b9ff2a0c3bb7b4bc7a6/Screenshot%202024-08-13%20at%204.11.27%E2%80%AFPM.png" alt="Screenshot 2024-08-13 at 4.11.27 PM.png" />

<img width="737" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL1JWQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--63300e9390e8bc55f18236a3bb44ea696a890879/Screenshot%202024-08-13%20at%204.11.01%E2%80%AFPM.png" alt="Screenshot 2024-08-13 at 4.11.01 PM.png" />

**Comment by laflannery** [comment]
@mmiddaugh I believe these are the widgets that are not in our portfolio? CC: @jilladams @FranECross

**Comment by jilladams** [comment]
Ownership hasn't transferred, but Michelle's working out details in hopes that we'll move parts: 
* Epic: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16547
* Scrum card: https://dsva.slack.com/lists/T03FECE8V/F079GFKN50D?record_id=Rec07A9MB02SW

**Comment by outerpress** [comment]
Thanks @jilladams and @laflannery (and sorry for the slow reply). So it sounds like it's not clear yet which team will own the widget. I'll keep an eye on the epic you posted for updates, and we'll keep this as blocked on our side. FYI @ldelacosta
